src/resources/locales/eu.json

```diff
@@ -0,0 +1,393 @@
+        },
+        "media-overlays": {
+            "activate": "Aktibatu ozenki irakurtzea",
+            "captions": "Testu sinkronizatua/aurkezpen diafanoa",
+            "next": "Hurrengoa (ozenki irakurtzea)",
+            "pause": "Geldiarazi (ozenki irakurri)",
+            "play": "Erreproduzitu (ozenki irakurtzea)",
+            "previous": "Aurrekoa (ozenki irakurtzea)",
+            "skip": "Ez jarri informazio osagarririk",
+            "speed": "Ozen irakurtzeko erreprodukzio-abiadura",
+            "stop": "Gelditu ozenki irakurtzea",
+            "title": "Ozenki irakurtzea"
+        },
+        "navigation": {
+            "backHomeTitle": "Itzuli liburutegira",
+            "bookmarkTitle": "Markatzailea",
+            "currentPage": "Uneko orrialdea: {{- current}}",
+            "currentPageTotal": "Uneko orrialdea: {{- current}}/{{- total}}",
+            "detachWindowTitle": "Liburutegiko leihoa erakutsi",
+            "fullscreenTitle": "Pantaila osoaren modua ireki",
+            "goTo": "Joan hona",
+            "goToError": "Orrialde-jauzi hori ez dago",
+            "goToPlaceHolder": "Sartu orrialde-zenbaki bat",
+            "goToTitle": "Orrialdera joan",
+            "infoTitle": "Informazioa",
+            "magnifyingGlassButton": "bilatu argitalpenean",
+            "openTableOfContentsTitle": "Nabigazioa",
+            "pdfscalemode": "Orriaren doikuntza edo zabalera",
+            "quitFullscreenTitle": "Pantaila osoko modutik ateratzea",
+            "readBookTitle": "Hasi liburua irakurtzen",
+            "settingsTitle": "Konfigurazio-aukerak"
+        },
+        "picker": {
+            "annotationTitle": "Oharrak:",
+            "search": {
+                "founds": "{{- nResults}} aurkitutako",
+                "input": "bilaketa-testua",
+                "next": "Hurrengoa",
+                "notFound": "ez dago bat etortzerik",
+                "previous": "Aurrekoa",
+                "submit": "Aurkitu"
+            },
+            "searchTitle": "Bilatu:"
+        },
+        "settings": {
+            "column": {
+                "auto": "Automatiko",
+                "one": "1 zut.",
+                "oneTitle": "Zutabe bat",
+                "title": "Zutabeak",
+                "two": "2 zutabeak",
+                "twoTitle": "Bi zutabe"
+            },
+            "display": "Bistaratzea",
+            "disposition": {
+                "title": "Xedapena"
+            },
+            "font": "Iturria",
+            "fontSize": "Letra-tamaina",
+            "justification": "Lerrokadura",
+            "justify": "Justifikatu",
+            "letterSpacing": "Letren tartea",
+            "lineSpacing": "Lerroartea",
+            "margin": "Marjina",
+            "noFootnotes": "Oin-oharrak desaktibatu",
+            "paginated": "Orrazkera",
+            "paraSpacing": "Paragrafoen arteko tartea",
+            "pdfZoom": {
+                "name": {
+                    "100pct": "100%",
```

Comment:
All percentages should have the % first (%100 as opposed to 100%). Please change this line and subsequent ones too.